README.md

```diff
@@ -37,17 +39,20 @@ file. Here are the steps to configure shadow flows:
 
 1. Define the shadow flows you want to use in your application under the `shadowflow.flows` property.
    In this example, `flow1` and `flow2` are defined as shadow flows (which you can rename to anything).
-   And `percentage` represents a percentage number in the range of 0-100 of how many calls should be compared in the
+   `percentage` represents a percentage number in the range of 0-100 of how many calls should be compared in the
    shadow flow.
    Zero effectively disables the shadow flow.
+   And `type` represents the data model which is used to compare.
```

Comment:
```suggestion
   shadow flow. \
   Zero effectively disables the shadow flow. \
   And `type` represents the data model which is used to compare.
```

Or did you mean to put it display it on one line?

Reply:
Yeah, I meant it to be displayed on one line. I added the return for readability in IDEs